src/__test__/actions/assetLogs.actions.test.js

```diff
@@ -0,0 +1,76 @@
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import constants from '../../_constants';
+
+import loadAssetLogs from '../../_actions/assetLogs.actions';
+
+import assetLogs from '../../_mock/assetLogs';
+
+const {
+  LOAD_ASSET_LOGS_SUCCESS,
+  LOAD_ASSET_LOGS_FAILURE,
+  LOADING_ASSETS_LOGS
+} = constants;
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+let store;
+let mock;
+
+describe('Asset Logs Actions', () => {
+  beforeEach(() => {
+    mock = new MockAdapter(axios);
+    store = mockStore({});
+  });
+
+  afterEach(() => {
+    store.clearActions();
+  });
+
+  it('should dispatch LOADING_ASSET_LOGS with isLoading false when done fetching asset logs', () => {
```

Comment:
For jest async tests, you should consider using `expect.hasAssertions` or similar to ensure it fails if assertions don't fire